lnwire/typed_fee.go

```diff
@@ -0,0 +1,60 @@
+package lnwire
+
+import (
+	"io"
+
+	"github.com/lightningnetwork/lnd/tlv"
+)
+
+const (
+	FeeRecordType tlv.Type = 55555
+)
+
+// Fee represents a fee schedule.
+type Fee struct {
```

Comment:
Non-blocking given the age/lineage of this PR, but I think we should go ahead and add it to the existing `ChannelUpdate` struct using the new optional TLV records.

Reply:
Will make a follow up issue to track this.

Reply:
Sounds good